scripts/build-bundle

```diff
@@ -9,6 +9,12 @@ PACKAGE_HARVESTER_REPO_DIR="${TOP_DIR}/package/harvester-repo"
 
 cd ${TOP_DIR}
 
+addons_path=../addons
+if [ ! -d ${addons_path} ];then
+    echo "No existed harvester source. Pulling..."
+    git clone --branch main --single-branch --depth 1 https://github.com/harvester/addons.git ../addons
+fi
+
```

Comment:
Do we really need this, given it's already done in `scripts/build`? (I assume the answer is "yes", given we already do the same with the harvester source, but I've just noticed the duplication, so I had to ask...)

Reply:
I believe both are around in case someone directly calls `make build` and `make build-bundle` targets manually